lib/import.js

```diff
@@ -149,6 +122,12 @@ class Import {
                 options.ignoreDup = !!options['ignore-dup'];
                 options.dryrun = !!options.dryrun;
 
+                if (argv.input) {
```

Comment:
It's strange to check `argv.input` and then use `options.input` - any reason for that?

Reply:
Yes, argv is the response from the promp library, options is what is actually used to make the request

Reply:
Good catch